notifier/plotting.go

```diff
@@ -32,9 +32,26 @@ func (err errFetchAvailableSeriesFailed) Error() string {
 	return fmt.Sprintf("Failed to fetch both realtime and stored data: [realtime]: %s, [stored]: %s", err.realtimeErr, err.storedErr)
 }
 
-// buildNotificationPackagePlot returns bytes slice containing package plot
-func (notifier *StandardNotifier) buildNotificationPackagePlots(pkg NotificationPackage) ([][]byte, error) {
+// buildTriggerPlots returns bytes slices containing trigger plots
+func buildTriggerPlots(trigger *moira.Trigger, metricsData map[string][]metricSource.MetricData, plotTemplate *plotting.Plot) ([][]byte, error) {
 	result := make([][]byte, 0)
+	for targetName := range metricsData {
+		metrics := metricsData[targetName]
```

Comment:
It's better to use for targetName, metrics := range metricsData here

Reply:
done